src/resources/i18n/translations/en.json

```diff
@@ -710,22 +710,21 @@
   "Use custom web domain": "Custom web domain",
   "Use External Storage For Downloads": "To external storage card",
   "Username": "Username",
-  "value_tag_preview_boost": "A boost sends a donation.",
-  "value_tag_preview_boostagram": "A boostagram also adds a public message.",
-  "value_tag_preview_streaming": "Stream sends a donation per 1 minute of listening time.",
+  "value_tag_preview_boost": "A <i>boost</i> sends a donation.",
+  "value_tag_preview_boostagram": "A <i>boostagram</i> also adds a public message.",
+  "value_tag_preview_streaming": "<i>Stream</i> sends a donation for every minuted listened.",
   "value_tag_preview_title": "Send Bitcoin to your favorite creators",
-  "value_tag_info_streaming_sats_text_1": "When enabled, for every minute of playback, transactions are added to your streaming sats queue for each recipient in the creator's value splits.",
-  "value_tag_info_streaming_sats_text_2": "Each minute, the streaming sats queue is processed, and each recipient that should receive 5 or more sats is sent their sats.",
-  "value_tag_info_streaming_sats_text_3": "If a recipient's sats total does not yet add up to 5 sats, they will continue accumulating in the queue until it adds up to 5 or more sats.",
-  "value_tag_info_streaming_sats_text_4": "Every time the app is closed and opened again, the streaming sats queue is cleared.",
-  "value_tag_info_streaming_sats_title": "About streaming sats",
-  "value_tag_streaming_sats": "Streaming sats",
+  "value_tag_info_streaming_sats_text_1": "A queue of transactions to recipients of the crator\'s choosing.",
+  "value_tag_info_streaming_sats_text_2": "Every minute of playback, each recipient gets their Sats once it adds up to 5.",
```

Comment:
There appears to be no hard consensus on whether it is "SATS" or "Sats", but it isn't "sats".
Since it is "Satoshis", I think "Sats" is better.

Reply:
"Sats" looks good.